lib/services/messenger/translators/translator-scaffold.ts

```diff
@@ -27,6 +27,10 @@ import {
 	ResponseConverters, TranslatorErrorCode, TranslatorMetadata,
 } from './translator-types';
 
+export enum MetadataEncoding {
+	hiddenHTML, hiddenMD, char,
```

Comment:
Can we have these starting with a Capital? It's easier to differentiate them from locally scoped vars, then.
Also, I'd strongly consider renaming `char`, which whilst not a reserved keyword in TS, is for future use in JS.